docs/source/run-workflows/observe/observe-workflow-with-phoenix.md

```
@@ -15,26 +15,42 @@ See the License for the specific language governing permissions and
 -->
 
-# Observing a Workflow with Phoenix
+# Observing a Workflow with Arize Phoenix
 
-This guide provides a step-by-step process to enable observability in a NeMo Agent Toolkit workflow using Phoenix for tracing and logging. By the end of this guide, you will have:
+This guide provides a step-by-step process to enable observability in a NeMo Agent Toolkit workflow using Arize Phoenix for tracing and logging. By the end of this guide, you will have:
 - Configured telemetry in your workflow.
 - Started the Phoenix server locally.
 - Ability to view traces in the Phoenix UI.
 
 ### Step 1: Install the Phoenix Subpackage and Phoenix Server
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)